whatlies/embeddingset.py

```diff
@@ -181,6 +181,15 @@ def compare_against(self, other, mapping="direct"):
         if mapping == "direct":
             return [v > other for k, v in self.embeddings.items()]
 
+    def pipe(self, func, *args, **kwargs):
+        """
+        Applies a function to the embedding set. Useful for method chaining.
+        :param func: The function to apply
```

Comment:
Has the style of docstrings changed? E.g. we didn't use `:param` previously?!

Reply:
Well spotted. Will fix this!